gensim/models/keyedvectors.py

```diff
@@ -1052,15 +1193,16 @@ def _log_evaluate_word_analogies(section):
             logger.info("%s: %.1f%% (%i/%i)", section['section'], 100.0 * score, correct, correct + incorrect)
             return score
 
-    def evaluate_word_analogies(self, analogies, restrict_vocab=300000, case_insensitive=True, dummy4unknown=False):
+    def evaluate_word_analogies(self, analogies, restrict_vocab=300000, case_insensitive=True,
+    dummy4unknown=False, topk=5, method='3CosAdd'):
```

Comment:
Why is `topk=5` the default? AFAIK, the analogy is a success only if it matches the top-1 word.

Reply:
At line 1081 of the existing code the given input in most_similar method is topn = 5. 
This is what i mean as the default value, which has been retained in my implementation.